packages/fx-core/src/common/projectSettingsHelper.ts

```diff
@@ -16,6 +16,8 @@ import {
 } from "../plugins/solution/fx-solution/question";
 import { BuiltInFeaturePluginNames } from "../plugins/solution/fx-solution/v3/constants";
 import * as uuid from "uuid";
+import { NotificationOptionItem } from "../core/question";
+import { isBotNotificationEnabled } from ".";
```

Comment:
from a real file to avoid dependency cycle.

Reply:
fixed